shell/platform/darwin/ios/framework/Source/FlutterPlatformPluginTest.mm

```diff
@@ -368,4 +368,93 @@ - (void)testViewControllerBasedStatusBarHiddenUpdate {
   [bundleMock stopMocking];
 }
 
+- (void)testStatusBarHiddenUpdate {
+  id bundleMock = OCMPartialMock([NSBundle mainBundle]);
+  OCMStub([bundleMock objectForInfoDictionaryKey:@"UIViewControllerBasedStatusBarAppearance"])
+      .andReturn(@NO);
+  id mockApplication = OCMClassMock([UIApplication class]);
+  OCMStub([mockApplication sharedApplication]).andReturn(mockApplication);
+
+  // Enabling system UI overlays to update status bar.
+  FlutterEngine* engine = [[FlutterEngine alloc] initWithName:@"test" project:nil];
```

Comment:
This file isn't ARC on `stable`, so all the `alloc` calls need `autorelease`.

Reply:
done